wp/lib/bap_wp/tests/unit/test_constraint.ml

```diff
@@ -113,9 +113,18 @@ let test_substitute_order (test_ctx : test_ctxt) : unit =
   let goals = Constr.get_refuted_goals pre solver ctx in
   assert_refuted_goals test_ctx post_expr goals
 
+let test_expr_to_hex (test_ctx : test_ctxt) : unit =
+  let ctx = Env.mk_ctx () in
+  let expr = BV.mk_numeral ctx "1073741824" 64 in
+  let result = Constr.expr_to_hex expr in
+  let expected = "0x0000000040000000" in
```

Comment:
Any chance we can get an `0xffffffffffffffff` example?

Reply:
Added that test and it does overflow. Going to fix that now.